testing/acceptance_test.py

```diff
@@ -1584,3 +1584,27 @@ def test_no_terminal_plugin(pytester: Pytester) -> None:
     pytester.makepyfile("def test(): assert 1 == 2")
     result = pytester.runpytest("-pno:terminal", "-s")
     assert result.ret == ExitCode.TESTS_FAILED
+
+def test_get_exception_on_teardown_failure(pytester: Pytester) -> None:
+    """Smoke test to be sure teardown exceptions handled properly via node property"""
+    pytester.makepyfile(
+        conftest="""
+        import sys
+        import pytest
+        def pytest_exception_interact(node, call, report):
+            sys.stderr.write("{}".format(node.teardown_exceptions))
+
+        import pytest
+        @pytest.fixture
+        def mylist():
+            yield
+            raise AssertionError(111)
+        """,
+        test_file="""
+        def test_func(mylist):
+            assert True
+        """,
+    )
+    result = pytester.runpytest()
+    assert result.ret == ExitCode.TESTS_FAILED
```

Comment:
There is no clear standard what method should be used for checking test passing counts. There is a `res.stdout.fnmatch_lines(["*1 passed*"])` and also a `res.assert_outcomes(failed=1)`. We probably should add one of these - either is fine

Reply:
Stick with `result.stdout.fnmatch_lines(["*1 error*"])` because unresolved https://github.com/pytest-dev/pytest/issues/9909